py/client/pydeephaven/session.py

```diff
@@ -414,12 +414,14 @@ def bind_table(self, name: str, table: Table) -> None:
         with self._r_lock:
             self.console_service.bind_table(table=table, variable_name=name)
 
-    def time_table(self, period: int, start_time: int = None) -> Table:
+    def time_table(self, period: Union[int, str], start_time: Union[int, str] = None) -> Table:
         """Creates a time table on the server.
 
         Args:
-            period (int): the interval (in nano seconds) at which the time table ticks (adds a row)
-            start_time (int): the start time for the time table in nano seconds, default is None (meaning now)
+            period (Union[int, str]): the interval at which the time table ticks (adds a row); units are nanoseconds
+                or a time interval string, e.g. "PT00:00:.001" or "PT1S"
+            start_time (Union[int, str]): the start time for the time table in nanoseconds or as an ISO 8601 date time
+                formatted string; default is None (meaning now)
```

Comment:
Are we really restricted to 8601, or do we support any format that the time library on the server can handle?  I'm guessing the latter.

Reply:
Looking later in the code, we do support anything that the time library can parse.

Reply:
Removed the 8601-specific text.